Google Play Music/CoreMusicApp.Hooks.cs

```diff
@@ -88,7 +88,12 @@ void gkh_KeyUp(object sender, KeyEventArgs e)
                         zoomInProgress = false;
                     }
                     break;
-
+                case Keys.F:
+                    if((ModifierKeys & Keys.Control) == Keys.Control)
```

Comment:
Also although looking at it `ModifierKeys` will do the same job as my `controlDown` variable can we please be consistent, so either change yours to match mine, or change mine to match yours.  I don't mind either way as long as it works